apps/site/redirects.json

```diff
@@ -8,6 +8,14 @@
       "source": "/api.html",
       "destination": "/api"
     },
+    {
+      "source": "/api/corepack.html",
```

Comment:
The redirect, if implemented for latest, should be done after corepack is not available in that Node.js version
i.e. when v25.0.0 is released.

While corepack is included in the latest, it should highlight the decision in bold and provide a call to action, like it's suggested in https://github.com/nodejs/node/pull/57825

Reply:
Whatever call to action you'd like to see, why does it need to be maintained in nodejs/node rather than nodejs/corepack?

Reply:
The call to action is for existing Node.js versions which ship corepack.
It's similar to [punycode](https://nodejs.org/api/punycode.html) which was deprecated since v7.0.0 but call to action still exists.

I don't suggest keeping corepack call to action forever. It needs to be minimally in Node.js versions which ship experimental corepack, i.e. v24. It'll be helpful if the call to action is in latest doc till <=v28. But dropping it in v27 is also okay.

Other options are mentioned in https://github.com/nodejs/node/pull/57663#issuecomment-2766906437

Reply:
Dismissed my review, and provided summary https://github.com/nodejs/nodejs.org/pull/7643#event-17240639556